### ISSUE TITLE
Achievements don't update on activity deletion

### ISSUE THREAD

**Issue by jibeetz** [peregin/velocorner.com#10]
Hi peregin!
The title says all. The achievements helped me identify an outlier, but they didn't update when I deleted the activity on Strava.
Thanks!

**Comment by peregin** [comment]
good catch @jibeetz , this is a bug indeed, synchronisation and aggregation ignores deletions.
